### ISSUE TITLE
Cannot setup Group chat

### ISSUE THREAD

**Issue by shejinbasheer01** [pazznetwork/ngx-chat#8]
I am Trying to include group chat. but my room array is always empty. Please help me on this. I can access my rooms via other xmpp clients. But here it is not loading.

**Comment by trampi** [comment]
Hello @shejinbasheer01,
at the moment, ngx-chat does not fetch / join Rooms automatically.

To join a room, you can do the following:
```typescript
const multiUserChat = chatService.getPlugin(MultiUserChatPlugin)
await multiUserChat.joinRoom('room@server/nickname')
```

To automatically join your favorite rooms after login, you can use the `BookmarkPlugin` to store / load the rooms to join, e.g. via:
```typescript
// joining rooms automatically
const bookmarkPlugin = chatService.getPlugin(BookmarkPlugin);
chatService.state$.pipe(filter(state => state === 'online')).subscribe(() => {
    const rooms = await bookmarkPlugin.retrieveMultiUserChatRooms();
    for (const room of rooms) {
        if (room.autoJoin) {
            await multiUserChat.joinRoom(room.jid);
        }
    }
});
```

To automatically save your favorite rooms after joining a room, you could do this:
```typescript
// saving joined rooms
const multiUserChat = chatService.getPlugin(MultiUserChatPlugin);
const bookmarkPlugin = chatService.getPlugin(BookmarkPlugin);
multiUserChat.rooms$.subscribe(rooms => {
    for (room of rooms) {
        bookmarkPlugin.saveConference({
            name: room.name,
            jid: room.roomJid,
            autojoin: true
        })
    }
})
```

If this does not solve your question, do not hesitate to ask!

**Comment by jameskentTX** [comment]
@shejinbasheer01:

Remove mod_vcard module from ejabberd.yml

**Comment by shejinbasheer01** [comment]
thank you for your response

**Comment by shejinbasheer01** [comment]
Still i am having an issue
it is only fetching one group
my contacts are subscribed to at least 5 to 6 groups. But it is only fetching 1 group

**Comment by trampi** [comment]
Hi @shejinbasheer01,
I could reproduce it locally. I will investigate and provide feedback shortly.

**Comment by trampi** [comment]
Hi @shejinbasheer01,
I just released ngx-chat@0.3.4 and ngx-chat@0.4.0, both fixing the bug you reported.
Please note that 0.4.0 requires Angular 8. You can use 0.3.4 if you are not yet using Angular 8.

You need to slightly modify the code I've sent you:
```typescript
bookmarkPlugin.addConference({ // was previously saveConference
```

I am closing this issue, if you have further questions do no hesitate to reopen this issue or open a new issue.

All the best,
trampi
